test/pytest.ini

```diff
@@ -0,0 +1,20 @@
+[pytest]
+asyncio_mode = auto
+
+log_cli = true
+log_format = %(asctime)s.%(msecs)03d %(levelname)s>  %(message)s
+log_date_format = %H:%M:%S
+
+markers =
+    slow: tests that take more than 30 seconds to run
+    replication_factor: replication factor for RandomTables
+testpaths = test
```

Comment:
What does this line do?

Reply:
This is [configuration](https://docs.pytest.org/en/stable/reference/reference.html#confval-testpaths) for pytest where to search test for execution.

Reply:
How does having a testpaths=test inside test/pytest.ini? I'm still confused. If you're in the top-level Scylla directory, this test/pytest.ini will not be noticed. If you `cd test`, you're already there, so testpaths=test is not needed. What am I missing?

Reply:
deleted